react/components/FilterOptionTemplate.js

```diff
@@ -17,7 +18,11 @@ const CSS_HANDLES = [
   'filterIcon',
   'filterContent',
   'filterTemplateOverflow',
+  'seeMoreButton',
 ]
+
+const MAX_TRUNCATED_ITEMS = 10
```

Comment:
I think this could be a prop eventually, but being hardcoded is just fine for now